scrooge-generator/src/main/resources/javagen/service.java

```diff
@@ -91,3 +95,38 @@ public FinagledService (
   {{>finagleService}}
 {{/finagleServices}}
 }
+
+
+class TFCF<T> extends CompletableFuture<T> {
```

Comment:
Really sorry to comment here, but wasn't sure how else to contact folks: somehow I've been accidentally added to this repository and am getting code review emails. I'm guessing it's because my name is "Ben Hoyt" (username "benhoyt") and there's a "b-hoyt" on this project. My email address is on my website at http://benhoyt.com/ -- can you please remove me from this project?